modules/caddyhttp/caddyhttp.go

```diff
@@ -185,7 +185,21 @@ func (app *App) Provision(ctx caddy.Context) error {
 			if err != nil {
 				return fmt.Errorf("server %s: setting up server error handling routes: %v", srvName, err)
 			}
-			srv.errorHandlerChain = srv.Errors.Routes.Compile(emptyHandler)
```

Comment:
`emptyHandler` is already at the end of the chain, so do you think we could just replace `emptyHandler` here with your handler that you're currently appending down on lines 194-202? Maybe just pull it out to a new unexported variable next to `emptyHandler` called `emptyErrorHandler`.

Reply:
Done